### PR TITLE
Added minor fixes for supporting calmvm-380

### PR DESCRIPTION
**Pull request by nisarg131**

### CODE REVIEW

File: solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/post-migration-script.py [hunk 1]
```diff
@@ -346,12 +344,13 @@ def get_vm_source_dest_uuid_map():
...
                 step_uuid = step_execution_status_src["step_uuid"]
                 src_vm_uuid = step_execution_status_src["any_entity_reference_list"][0]["uuid"]
-                for step_execution_status_dest in step_execution_status_list:
-                    if (
-                        step_execution_status_dest["parent_step_uuid"] == step_uuid and
-                        step_execution_status_dest["operation_type"] in ["ENTITY_RESTORATION", "ENTITY_MIGRATION"]
-                    ):
-                        dest_vm_uuid = step_execution_status_dest["any_entity_reference_list"][0]["uuid"]
+                dest_vm_uuid = step_execution_status_src["recovered_entity_info_list"][0]["recovered_entity_info"].get("entity_uuid")
+                #for step_execution_status_dest in step_execution_status_list:
```
Comment (JonKohler) [inline solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/post-migration-script.py:348]: Why is this all commented out? Can we just remove it instead?

File: solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/post-migration-script.py [hunk 2]
```diff
@@ -373,4 +372,4 @@ def main():
         raise
 
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (JonKohler) [inline solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/post-migration-script.py:375]: Is this necessary?

File: solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/pre-migration-script.py [hunk 3]
```diff
@@ -172,4 +175,4 @@ def main():
         log.info("Exception: %s" % e)
         raise
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (JonKohler) [inline solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/pre-migration-script.py:178]: ditto

Comment (JonKohler) [reply solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/pre-migration-script.py:178]: The commits did not address this one and the previous main() comment. Trivial, but reducing churn is nice when we can get it

Comment (nisarg131) [reply solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/pre-migration-script.py]: Done.

**Review by JonKohler — APPROVED** [review]

**Pull request merged**